src/world/entity.rs

```diff
@@ -1,9 +1,9 @@
 use std::{
     fmt,
-    num::NonZeroI32,
     sync::atomic::{AtomicUsize, Ordering},
 };
 
+use nonzero_signed::NonZeroI32;
```

Comment:
I'm confused by this; it's now deprecated since std has support for it.

Reply:
the import was failing for me

Reply:
Have you updates your Rust toolchain?

Reply:
fixed thanks